packages/earn-controller/src/EarnController.ts

```diff
@@ -140,6 +156,7 @@ const earnControllerMetadata: StateMetadata<EarnControllerState> = {
 export type EarnControllerState = {
   pooled_staking: PooledStakingState;
```

Comment:
May need this to avoid breaking changes  `// eslint-disable-next-line @typescript-eslint/naming-convention`